ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Superuser/tasks/main.yml

```diff
@@ -4,6 +4,7 @@
 #############
 - name: Create Superuser account - zeus
   action: user name=zeus state=present shell=/bin/bash
```

Comment:
I would strongly advise not setting this to `/bin/bash` as it's not guaranteed to be there on a basic Solaris installation